src/solvers/smt2/smt2_conv.cpp

```diff
@@ -2671,7 +2684,8 @@ void smt2_convt::convert_constant(const constant_exprt &expr)
     assert(it!=defined_expressions.end());
     out << it->second;
   }
-  else if(expr_type.id()==ID_rational)
+  else if(expr_type.id()==ID_rational ||
+          expr_type.id()==ID_real)
```

Comment:
How would this code work for `ID_real` when below it searches for `/`?

Reply:
No, taking real out.